### PR TITLE
[123] Implement get report API @GET

### PR DESCRIPTION
**Pull request by badnameok**
## What did you do

- added jobTitle to getReports and removed createdAt field from individual reports

## Demo

- login on swagger
- use the get reports API with arbitrary jobID between 1-10

### CODE REVIEW

**Comment by pisuttun** [comment]
เติม `ReporterName`ด้วยๆ (ทำให้ตรงกับ​ schema สิ​ ;w;)

**Review by miello — APPROVED** [review]
LGTM, please wait for review from others cc. @PKhing, @palmcm 

@pisuttun please recheck schema again

**Review by pisuttun — APPROVED** [review]
LGTM 👍

**Pull request merged**
